tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -377,13 +424,18 @@ cost_decompress_sorted_merge_append(PlannerInfo *root, DecompressChunkPath *dcpa
 	 * point. So, we use a quadratic cost model here to have higher costs than the normal
 	 * decompression when more than ~100 batches are used. We set
 	 * DECOMPRESS_CHUNK_HEAP_MERGE_CPU_TUPLE_COST to 0.8 to become most costly as soon as we have to
```

Comment:
Since the statistics can be outdated: 

```suggestion
	 * DECOMPRESS_CHUNK_HEAP_MERGE_CPU_TUPLE_COST to 0.8 to become most costly as soon as we expect to
```